### DISCUSSION TITLE
proxy-cache plugin redis strategy

### DISCUSSION THREAD

**Discussion by flowdopip** [Kong/kong#11999] [Help] [answered]
Hi all,

I'm trying to extend the current proxy-cache plugin and support the Redis strategy. The current implementation is already prepared for strategies, but there´s no documentation on how to extend to Redis or other technology.

Can someone help?

https://github.com/Kong/kong/tree/master/kong/plugins/proxy-cache/strategies

Thanks

**Comment by guanlan** [comment, marked as answer]
We have the official support in the enterprise version, https://docs.konghq.com/hub/kong-inc/proxy-cache-advanced/.

However, a few similar plugin forks are available on GitHub.
